src/math/CMakeLists.txt

```diff
@@ -5,31 +5,41 @@ if(BUILD_LIBRARY)
 	return()
 endif()
 
+is_zephyr(it_is_zephyr)
+
 add_local_sources(sof numbers.c)
 
-if(CONFIG_CORDIC_FIXED)
+if(CONFIG_CORDIC_FIXED OR it_is_zephyr)
         add_local_sources(sof trig.c)
 endif()
 
 add_local_sources_ifdef(CONFIG_SQRT_FIXED sof sqrt_int16.c)
 
 add_local_sources_ifdef(CONFIG_MATH_EXP sof exp_fcn.c exp_fcn_hifi.c)
 
-if(CONFIG_MATH_DECIBELS)
+if(CONFIG_MATH_DECIBELS OR it_is_zephyr)
```

Comment:
Using the same name for managing math functions under Zephyr is not a good idea.

Reply:
my question of the day to these changes: I understand that it's just carried over from the present global sof/zephyr/CMakeLists.txt, but now that we move to these local files, shouldn't we rather just `select MATH_DECIBELS` somewhere with Zephyr?

Reply:
Yes, I am thinking about it, and it makes more sense to me to have a select option with Zephyr.